settings/provisioner/virsh/virsh.spec

```diff
@@ -8,28 +8,39 @@ subparsers:
                   host:
                       type: str
                       help: Address/FQDN of the BM hypervisor
+                      required: yes
                   ssh-user:
                       type: str
                       help: User to SSH to the host with
+                      default: root
+                      required: yes
```

Comment:
What's the point of using 'required' if 'default' is defined?
(Relevant for all options with 'required' & 'default' in this file)